packages/fxa-payments-server/src/components/ProviderDetails/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Localized } from '@fluent/react';
+import { Customer } from '../../store/types';
+import * as Provider from '../../lib/provider';
+
+import './index.scss';
+
+type ProviderDetailsProps = {
+  customer: Customer;
+};
+
+export const ProviderDetails = ({ customer }: ProviderDetailsProps) => {
+  const { brand, last4, payment_provider } = customer;
+
+  return (
+    <>
+      {Provider.isStripe(payment_provider) && brand && last4 && (
```

Comment:
I wish I can tell you (without digging through server code and possibly Stripe docs) that if Provider.isStripe then that means `brand` and `last4` exist.  But at least now we keep this logic in just this component.  👍